core/src/data/key.rs

```diff
@@ -84,6 +84,7 @@ impl TryFrom<Value> for Key {
             Uuid(v) => Ok(Key::Uuid(v)),
             Decimal(v) => Ok(Key::Decimal(v)),
             Null => Ok(Key::None),
+            F32(_) => Err(KeyError::FloatTypeKeyNotSupported.into()),
             F64(_) => Err(KeyError::FloatTypeKeyNotSupported.into()),
```

Comment:
Same error so we can merge this into same branch
`F32(_) | F64(_) => Err(KeyError::FloatTypeKeyNotSupported.into()),`